pkg/internal/tests/cross-tests/create.go

```diff
@@ -103,17 +106,49 @@ func Create(
 	assertCtyValEqual(t, "RawConfig", tfResult.data.GetRawConfig(), puResult.data.GetRawConfig())
 	assertCtyValEqual(t, "RawPlan", tfResult.data.GetRawPlan(), puResult.data.GetRawPlan())
 	assertCtyValEqual(t, "RawState", tfResult.data.GetRawState(), puResult.data.GetRawState())
+
+	for k := range resource {
+		// TODO: make this recursive
```

Comment:
What does this mean here? How can we get to 100%?

Reply:
If I remember right, we are comparing some eeky structures here coming from the TFSDK, these structures resist directly comparing for equality but this is something we need for the purposes of the tests. It is a little tedious to get right but sounds like it should be doable within an hour or two. Am I understanding right? Need a hand here perhaps?